src/txn/txn_rollback_to_stable.c

```diff
@@ -1623,6 +1623,18 @@ __rollback_progress_msg(WT_SESSION_IMPL *session, struct timespec rollback_start
     }
 }
 
+/*
+ * __check_rollback_to_stable_btree_dirty --
+ *     Check that the rollback to stable btree is dirty or not.
+ */
+static int
+__check_rollback_to_stable_btree_dirty(WT_SESSION_IMPL *session, const char *uri, bool *perform_rts)
```

Comment:
The naming of the function is a bit confusing. How about just calling it `__check_btree_dirty`?

Reply:
Given [this comment](https://github.com/wiredtiger/wiredtiger/pull/8239#discussion_r963251237), we agree on `__check_btree_modified`.